### DISCUSSION TITLE
Termostato

### DISCUSSION THREAD

**Discussion by 88Marius88** [indomus/forum#596] [GENERALE - Discussioni generiche]
Salve ,,a  tutti sono un neofita , sto cercando di impostare il termostato , ma mi restituisce dei valori sballati e non veritieri. Come posso sistemarlo?
![Cattura](https://user-images.githubusercontent.com/87854623/128638374-a1bbe96c-1409-4feb-8db4-715fe0fbb476.PNG)

**Comment by snakuzzo** [comment]
Sì ma senza scrivere come hai creato l'entità è difficile darti una mano

**Reply by 88Marius88** [reply]
L'entità e uscita fuori collegando a tuya. Almeno credo intendessi questo.

**Comment by rexer78** [comment]
https://www.home-assistant.io/integrations/tuya/#climate-options
Qui ci sono opzioni per modificare alcune impostazioni del climate

**Reply by 88Marius88** [reply]
Si ma non riesco a capire come modificare i file per far mostrare la temperatura corretta

**Comment by LoRacolo** [comment]
Sei poi riuscito a risolvere? Mi trovo in una situazione molto simile.

**Reply by 88Marius88** [reply]
Ciao si ho risolto.

**Reply by LoRacolo** [reply]
Mi potresti dire come?

**Comment by 88Marius88** [comment]
I passaggi: 1.- Vai a Configurazione → Integrazioni → Tuya → Configura 2.- Scegli il tuo termostato negli ultimi 2 spazi e il pulsante 'Invia'. 3.- Quindi metti le tue esigenze, se mostra il doppio, dividi per 2 e 'Metà' come nell'immagine (se HA mostra 10 volte il valore reale, quindi dividi per 10 e 'Decimi') 4.- Invia 5. - Invia di nuovo senza toccare nulla.

**Reply by snakuzzo** [reply]
Potreste evitare di creare sempre nuovi thread ?
Rispondete sotto l'ultimo messaggio.
Grazie

**Reply by LoRacolo** [reply]
> I passaggi: 1.- Vai a Configurazione → Integrazioni → Tuya → Configura 2.- Scegli il tuo termostato negli ultimi 2 spazi e il pulsante 'Invia'. 3.- Quindi metti le tue esigenze, se mostra il doppio, dividi per 2 e 'Metà' come nell'immagine (se HA mostra 10 volte il valore reale, quindi dividi per 10 e 'Decimi') 4.- Invia 5. - Invia di nuovo senza toccare nulla.

Grazie per la risposta, @88Marius88 , ho provato a fare quello che dici tu, ma non mi compare la voce "Configura" nel menù dell'integrazione Tuya.
![image](https://user-images.githubusercontent.com/44835819/140605042-5fa5e6cc-5dc3-457d-83e4-2f20c56a09c9.png)
da personalizzazioni, scegliendo l'unica entità generata dal dispositivo (di tipo climate), vedo:
![image](https://user-images.githubusercontent.com/44835819/140605178-66e86701-1821-49c3-8554-9d5514bc3877.png)
ma la current temp è costantemente null.
